cirq-google/cirq_google/ops/internal_tag.py

```diff
@@ -62,18 +60,28 @@ def _value_equality_values_(self):
         return (self.name, self.package, tag_args_eq_values)
 
     def to_proto(self, msg: Optional[program_pb2.Tag] = None) -> program_pb2.Tag:
+        # To avoid circular import
+        from cirq_google.serialization import arg_func_langs
+
         if msg is None:
             msg = program_pb2.Tag()
         msg.internal_tag.tag_name = self.name
         msg.internal_tag.tag_package = self.package
+        for k, v in self.tag_args.items():
+            arg_func_langs.arg_to_proto(v, out=msg.internal_tag.tag_args[k])
         return msg
 
     @staticmethod
     def from_proto(msg: program_pb2.Tag) -> 'InternalTag':
+        # To avoid circular import
+        from cirq_google.serialization import arg_func_langs
+
+        kw_dict = {}
+        for k, v in msg.internal_tag.tag_args.items():
+            kw_dict[k] = arg_func_langs.arg_from_proto(v, arg_function_language='exp')
```

Comment:
Should we set `arg_function_language` also in the `to_proto` method above at line 71?

Reply:
Done!